app/models/tag.rb

```diff
@@ -133,7 +133,8 @@ def find_or_create_by_names(name_or_names)
     def search_for(term, limit = 5, offset = 0, options = {})
       stripped_term = term.strip
 
-      query = Tag.listable.matches_name(stripped_term)
+      query = Tag.matches_name(stripped_term)
+      query = query.merge(Tag.listable) unless options[:include_unlistable]
```

Comment:
This change was necessary because in the Admin UI we want to see all hashtags when we search for a hashtag, so admin sets `include_unlistable` to true; I thought this was the cleaner way of handling this, instead of an `exclude_unlistable` and then trying to find all the code paths that used `search_for` to make them add that option.

Reply:
Having something like `{ exclude_unlistable: true }.merge(options)` would allow a `exclude_unlistable` option without changing the call sites, but I think `include_unlistable` is fine too.

Reply:
Oooh, that could be a good call. I think I like that better given there's already an exclude_... for something else here

Reply:
Do you want to make that change then? Personally I believe both approaches are perfectly fine, but I agree it could make for a nicer "symmetry" with `exclude_unreviewed`.

Reply:
Yeah, I will, have just been dealing with other things this week